instrumentation/grape/test/opentelemetry/instrumentation/grape_test.rb

```diff
@@ -128,7 +128,7 @@ class VersionedWithPrefixAPI < Grape::API
 
       let(:app) { VersionedWithPrefixAPI }
       let(:request_path) { '/api/v1/hello' }
-      let(:expected_span_name) { 'VersionedWithPrefixAPI GET /api/v1/hello' }
+      let(:expected_span_name) { 'GET /api/v1/hello' }
```

Comment:
I am curious if the version is substituted but not the route params when version is specified.

Reply:
ah sweet, yeah I was kind of concerned about that and just wanted to be sure the two were unrelated 🙈  👍